server/server.js

```diff
@@ -18,54 +19,54 @@ app.get("/", (req, res) => {
 
 //create the get request
 app.get("/api/books", cors(), async (req, res) => {
-  // const BOOKS = [
-  //   {
-  //     id: 1,
-  //     title: "Written in Red",
-  //     author_f: "Anne",
-  //     author_l: "Bishop",
-  //     format: "Book",
-  //     owned: "false",
-  //     read: "true",
-  //   },
-  //   {
-  //     id: 2,
-  //     title: "Written in Red",
-  //     author_f: "Anne",
-  //     author_l: "Bishop",
-  //     format: "Audiobook",
-  //     owned: "false",
-  //     read: "true",
-  //   },
-  //   {
-  //     id: 3,
-  //     title: "The Diamond Age: Or, a Young Lady's Illustrated Primer",
-  //     author_f: "Neal",
-  //     author_l: "Stephenson",
-  //     format: "Book",
-  //     owned: "true",
-  //     read: "true",
-  //   },
-  // {
-  //   id: 4,
-  //   title: "",
-  //   author_f: "",
-  //   author_l: "",
-  //   format: "",
-  //   owned: "",
-  //   read: "",
-  // },
-  // {
-  //   id: 5,
-  //   title: "",
-  //   author_f: "",
-  //   author_l: "",
-  //   format: "",
-  //   owned: "",
-  //   read: "",
-  // },
-  // ];
-  // res.json(BOOKS);
+  const mockBooks = [
+    {
+      id: 1,
+      title: "Written in Red",
+      author_f: "Anne",
+      author_l: "Bishop",
+      format: "Book",
+      owned: "false",
+      read: "true",
+    },
+    {
+      id: 2,
+      title: "Written in Red",
+      author_f: "Anne",
+      author_l: "Bishop",
+      format: "Audiobook",
+      owned: "false",
+      read: "true",
+    },
+    {
+      id: 3,
+      title: "The Diamond Age: Or, a Young Lady's Illustrated Primer",
+      author_f: "Neal",
+      author_l: "Stephenson",
+      format: "Book",
+      owned: "true",
+      read: "true",
+    },
+    {
+      id: 4,
+      title: "",
+      author_f: "",
+      author_l: "",
+      format: "",
+      owned: "",
+      read: "",
+    },
+    {
+      id: 5,
+      title: "",
+      author_f: "",
+      author_l: "",
+      format: "",
+      owned: "",
+      read: "",
+    },
+  ];
+  // res.json(mockBooks);
```

Comment:
remove?

Reply:
The mockData and response or just the response?